v2/component/runtime_info.go

```diff
@@ -85,3 +83,250 @@ func parseRuntimeInfo(jsonEncoded string) (*runtimeInfo, error) {
 
 	return r, nil
 }
+
+func pipelineSpecValuetoMLMDValue(v *pipeline_spec.Value) (*pb.Value, error) {
```

Comment:
nit: `pipelineSpecValueToMLMDValue`

Note `To`

Reply:
Done.